src/pages/public_registration/action_accordions/MoreActionsCollapse.tsx

```diff
@@ -0,0 +1,45 @@
+import ExpandLess from '@mui/icons-material/ExpandLess';
+import ExpandMore from '@mui/icons-material/ExpandMore';
+import { Box, Divider, IconButton } from '@mui/material';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Registration } from '../../../types/registration.types';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { TerminateRegistration } from './TerminateRegistration';
+import { UnpublishRegistration } from './UnpublishRegistration';
+
+interface MoreActionsCollapseProps {
+  registration: Registration;
+}
+
+export const MoreActionsCollapse = ({ registration }: MoreActionsCollapseProps) => {
```

Comment:
Usikker på naming her 🤔 Landet på dette fordi den er collapsible, og at den innehar flere actions (avpubliser, sletting, republiser). Men syns ikke navnet ble veldig intuitivt, så åpen for andre forslag

Reply:
Nå vurderte jeg ikke navnet før jeg leste forklaringen din, så jeg syns navnet er intuitivt!

Reply:
Eventuelt MoreActionsCollapsible